# 1.Import necessary packages

In [1]:
import pandas as pd
import numpy as np
import seaborn as sns
import matplotlib.pyplot as plt
sns.set(color_codes=True)
%matplotlib inline

# 2.Load the file

In [5]:
df1 = pd.read_csv('Inc_Exp_Data.csv')
df1

Mthly_HH_Income  Mthly_HH_Expense  No_of_Fly_Members  Emi_or_Rent_Amt  \
0              5000              8000                  3             2000   
1              6000              7000                  2             3000   
2             10000              4500                  2                0   
3             10000              2000                  1                0   
4             12500             12000                  2             3000   
5             14000              8000                  2                0   
6             15000             16000                  3            35000   
7             18000             20000                  5             8000   
8             19000              9000                  2                0   
9             20000              9000                  4                0   
10            20000             18000                  4             8000   
11            22000             25000                  6            12000   
12            23400              5000                  3                0   
13            24000             10500                  6                0   
14            24000             10000                  4                0   
15            25000             12300                  3                0   
16            25000             20000                  3             3500   
17            25000             10000                  6                0   
18            29000              6600                  2             2000   
19            30000             13000                  4                0   
20            30500             25000                  5             5000   
21            32000             15000                  4                0   
22            34000             19000                  6                0   
23            34000             25000                  3             4000   
24            35000             12000                  3                0   
25            35000             25000                  4                0   
26            39000              8000                  4                0   
27            40000             10000                  4                0   
28            42000             15000                  4                0   
29            43000             12000                  4                0   
30            45000             25000                  6                0   
31            45000             40000                  6             3500   
32            45000             10000                  2             1000   
33            45000             22000                  4             2500   
34            46000             25000                  5             3500   
35            47000             15000                  7                0   
36            50000             20000                  4                0   
37            50500             20000                  3                0   
38            55000             45000                  6            12000   
39            60000             10000                  3                0   
40            60000             50000                  6            10000   
41            65000             20000                  4             5000   
42            70000              9000                  2                0   
43            80000             20000                  4                0   
44            85000             25000                  5                0   
45            90000             48000                  7                0   
46            98000             25000                  5                0   
47           100000             30000                  6                0   
48           100000             50000                  4            20000   
49           100000             40000                  6            10000   

    Annual_HH_Income Highest_Qualified_Member  No_of_Earning_Members  
0    

# 3.Analyze the data

In [6]:
df1.head()

Mthly_HH_Income  Mthly_HH_Expense  No_of_Fly_Members  Emi_or_Rent_Amt  \
0             5000              8000                  3             2000   
1             6000              7000                  2             3000   
2            10000              4500                  2                0   
3            10000              2000                  1                0   
4            12500             12000                  2             3000   

   Annual_HH_Income Highest_Qualified_Member  No_of_Earning_Members  
0             64200           Under-Graduate                      1  
1             79920               Illiterate                      1  
2            112800           Under-Graduate                      1  
3             97200               Illiterate                      1  
4            147000                 Graduate                      1

# 4.What is the Mean Expense of a Household?

In [9]:
df1['Mthly_HH_Expense'].mean()

18818.0

# 5.What is the Median Household Expense?

In [10]:
df1['Mthly_HH_Expense'].median()

15500.0

# 6.What is the Monthly Expense for most of the Households?

In [12]:
df1['Mthly_HH_Expense'].mode()

0    25000
dtype: int64

# 7.Plot the Histogram to count the Highest qualified member

# 8.Calculate IQR(difference between 75% and 25% quartile)

# 9.Calculate Standard Deviation for first 4 columns.

# 10.Calculate Variance for first 3 columns.

# 11.Calculate the count of Highest qualified member.

# 12.Plot the Histogram to count the No_of_Earning_Members

# 13.Suppose you have option to invest in Stock A or Stock B. The stocks • have different expected returns and standard deviations. The expected return of Stock A is 15% and Stock B is 10%. Standard Deviation of the returns of these stocks is 10% and 5% respectively.

Which is better investment?